### PR TITLE
Separate Planetiler-core LangUtils and OmtLangUtils

### PR DESCRIPTION
**Pull request by lazaa32**

### CODE REVIEW

File: src/main/java/com/onthegomap/planetiler/openmaptiles/util/Utils.java [hunk 1]
```diff
@@ -1,78 +0,0 @@
-package com.onthegomap.planetiler.openmaptiles.util;
```
Comment (msbarry) [inline src/main/java/com/onthegomap/planetiler/openmaptiles/util/Utils.java:1]: we ended up not moving utils to core, let's revert this change which should fix the tests as well.

Comment (lazaa32) [reply src/main/java/com/onthegomap/planetiler/openmaptiles/util/Utils.java]: @msbarry Reverted. It passed the tests.
